docs/decisions/20230928-revocation-status-list-expansion-strategy.md

```diff
@@ -0,0 +1,74 @@
+# JWT credential revocation status list expansion strategy 
+
+- Status: accepted
+- Decider: Benjamin Voiturier, Yurii Shynbuiev, Ezequiel Postan, Shota Jolbordi
+- Date 2023-09-28
+- Tags: revocation, revocation-status-list, jwt-vs-revocation, statusList2021
+
+Technical Story: [Revocation status list expansion strategy decision | https://input-output.atlassian.net/browse/ATL-6012]
+
+## Context and Problem Statement
+
+In the process of devising a mechanism for revoking JWT credentials, we've opted to implement the [statusList2021](https://www.w3.org/TR/vc-status-list/) method. 
+This approach entails the creation of a "status list credential" that incorporates a gZip compressed status list in a form a bitString.
+The specification recommends a minimum size of 16 KB for the status list included in a credential.
+However, it does not delineate a maximum size, nor does it provide guidance on how to proceed if the selected status list surpasses its capacity to store information about revoked credentials.
+Put differently, if more credentials are issued than can be accommodated by a 16 KB status list, no specific instructions are provided.
+
+
+## Decision Drivers
```

Comment:
I suggest switching between the different storages (aka. VDR abstraction) to add to the decision drivers.

For the first option (double the size), there will be a point in time when the list is too extensive to be stored in the blockchain.  So, theoretically, a particular blockchain will limit the maximum size of the revocation list. It also depends on the implementation, for instance, we can store only a hash of the revocation list.
For the second option, the size of the revocation list is under our control.